features/steps/shared/dashboard.rb

```diff
@@ -0,0 +1,21 @@
+module SharedDashboard
+  include Spinach::DSL
+
+  step 'I click "Authored by me" link' do
+    within ".assignee-filter" do
+      click_link "Any"
+    end
+    within ".author-filter" do
+      click_link current_user.name
+    end
+  end
+
+  step 'I click "All" link' do
+    within ".author-filter" do
+      click_link "Any"
+    end
+    within ".assignee-filter" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.